examples/audit-croatia/audit.js

```diff
@@ -0,0 +1,78 @@
+function abortAudit(message) {
+    Output.renderText("Please try again by following the instructions in the script.");
+    return AuditResult.aborted(message);
+}
+
+function abortOrInvalidateAudit(message) {
+    if (Input.readBoolean("Should the audit of this asset be marked as failed?")) {
+        return AuditResult.failure(message);
+    }
+
+    Output.renderText("Please try again by following the instructions in the script.");
+    return AuditResult.aborted(message);
+}
+
+function audit(auditData) {
+    Output.renderHtml(Ipfs.getFile("step-1.html"));
+    Input.button("Continue");
+    Output.renderHtml(Ipfs.getFile("step-2.html"));
+
+    if (!Input.readBoolean("Is everything as described above?")) {
+        return abortAudit("Audit aborted before validation");
+    }
+
+    Output.renderHtml(Ipfs.getFile("step-3.html"));
+
+    if (!Input.readBoolean("Does the search prompt look like described above?")) {
+        return abortAudit("Audit aborted before cadastral office was found");
+    }
+
+    Output.renderHtml(
+        Ipfs.getFile("step-4.html")
+            .replace(/{cadastralOffice}/g, auditData.assetInfo.cadastralOffice)
```

Comment:
Can you, please, explain '/g'?

Reply:
This is JavaScript's regex syntax: `/regex/g` - `g` is a global flag which basically means "replace all". If I used `/regex/` then it would only replace the first match of the regex. This is similar to how `sed` command line utility regex syntax works. I tried using `replaceAll` but it is not available in the current JS runtime provided by Graal.